dbr_test.go

```diff
@@ -174,15 +176,19 @@ func TestTimeout(t *testing.T) {
 		var people []dbrPerson
 		_, err := sess.Select("*").From("dbr_people").Load(&people)
 		require.Equal(t, context.DeadlineExceeded, err)
+		assert.Equal(t, 1, sess.EventReceiver.(*testTraceReceiver).errored)
```

Comment:
could you use `require` package instead?

Reply:
Sure, one sec

Reply:
I have changed it. However in general I like to use `require` only for things that will prevent a panic if the test proceeds, and `assert` for things where it is OK to keep running.

Reply:
could you fix asserts in other places as well? thanks!
It is mostly for style consistency. We only use require, which stops the test execution if unexpected thing happens. The good part is that we don't have to think what will cause panic, and reduce 1 test dependency (instead of both `require` and `assert`).

Reply:
Sure thing, all done. Thanks!